### PR TITLE
WT-9064 Restore FLCS zero illusion for cursor->remove

### PR DESCRIPTION
**Pull request by sauclovian-wt**

### CODE REVIEW

**Comment by lukech** [comment]
autobuild: Can one of the admins verify this patch?

File: src/btree/bt_cursor.c [hunk 1]
```diff
@@ -1155,8 +1156,20 @@ __wt_btcur_remove(WT_CURSOR_BTREE *cbt, bool positioned)
...
+                 * To preserve the illusion that deleted values are 0 and that we can therefore
+                 * delete them, without violating the system restriction against consecutive
+                 * tombstones, generate a dummy value for the new tombstone to delete. Make the
+                 * dummy value zero for good measure.
+                 */
+                cbt->v = 0;
+                flcs_zero.data = &cbt->v;
+                flcs_zero.size = 1;
+                WT_ERR(__cursor_col_modify(cbt, &flcs_zero, WT_UPDATE_STANDARD));
+            }
```
Comment (keithbostic) [inline src/btree/bt_cursor.c:1172]: Can't we just do this with `static const WT_ITEM flcs_zero = { "", 1, NULL, 0, 0};`?

**Review by keithbostic — APPROVED** [review]

File: src/btree/bt_cursor.c (hunk 1 above)
Comment (sauclovian-wt) [reply src/btree/bt_cursor.c]: Wasn't sure if that was allowable. Anyway, yes (though it apparently can't be const); fixed in [0e146ad](https://github.com/wiredtiger/wiredtiger/pull/7746/commits/0e146ad09c260a84d4560fbef72def4cd789958c).

Comment (sauclovian-wt) [reply src/btree/bt_cursor.c]: Ah, it can be const, seems that __cursor_col_modify throws away __wt_col_modify's const. Fixing that...

Comment (sauclovian-wt) [reply src/btree/bt_cursor.c]: Done in [aa38f2e](https://github.com/wiredtiger/wiredtiger/pull/7746/commits/aa38f2e8bd66dbd8f4341021dc264ca0cf2c4456).

**Pull request merged**